### ISSUE TITLE
Keeps processes running after quitting

### ISSUE THREAD

**Issue by MennoLabs** [IsmaelMartinez/teams-for-linux#1532]
After quitting the application (from the tray), ps shows there are still processes running.

1. Start Teams and log in.
2. Log out and quit Teams (using menu in tray).
3. In a terminal type ps aux | grep teams | grep -v grep
4. Find three processes labeled  /snap/teams-for-linux/724/teams-for-linux.

**Expected behavior**
Quitting an application should remove all its processes IMHO.

**Desktop (please complete the following information):**
 - OS: Kubuntu 24.10
 - Installation package snap
 - Version 1.12.5
 

**Debug**
See attached file.
[teamsdebug.txt](https://github.com/user-attachments/files/18218425/teamsdebug.txt)

**Comment by IsmaelMartinez** [comment]
Hello @MennoLabs , 

You can use the `closeAppOnCross` if you want the application to quit on clicking the cross.

Thanks!

**Comment by MennoLabs** [comment]
That makes the icon in the tray disappear, but still leaves the processes.

**Comment by IsmaelMartinez** [comment]
well, then that is a bug if it doesn't close it.

**Comment by MennoLabs** [comment]
I tried running the .deb before and that would not launch.
For now I'll just use a quick bash script to kill the remaining processes.
And for the longer term I'm looking at moving away from (K)Ubuntu.

**Comment by IsmaelMartinez** [comment]
If you can make those amendments and run it with the command/logs, then I can have a further look.

**Comment by IsmaelMartinez** [comment]
Doesn't this link provide you with enough info? https://github.com/IsmaelMartinez/teams-for-linux/issues/1426#issuecomment-2384889316 . Is downloading a file, putting it into a location and running a command.

The logs look as clean as they should be, if you run it now with

```
teams-for-linux --disableGpu --logConfig='{}' --closeAppOnCross
```

Then when you click on the cross, it should kill all processes.

**Comment by MennoLabs** [comment]
Yes, thanks, this link provides enough info. The link from Dec 21 points somewhere else in that thread.
Still remaining processes after quitting though.

**Comment by MennoLabs** [comment]
Edit: they must have been leftover from an earlier launch.
It works now as expected, quitting kills all processes.
